extensions/ql-vscode/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Prints end-of-query evaluator log summaries to the Query Log. [#1349](https://github.com/github/vscode-codeql/pull/1349)
 - Be consistent about casing in Query History menu. [#1369](https://github.com/github/vscode-codeql/pull/1369)
+- Use base styling for MRVA results font [#1375](https://github.com/github/vscode-codeql/pull/1375)
```

Comment:
I don't think we're putting MRVA changes into the changelog since this feature is not yet publicly released. So, this line can be removed.